tests/test_latentdistributiontest.py

```diff
@@ -56,13 +57,43 @@ def test_bad_kwargs(self):
             LatentDistributionTest(size_correction=0)
         with self.assertRaises(TypeError):
             LatentDistributionTest(pooled=0)
+        with self.assertRaises(TypeError):
+            ldt = LatentDistributionTest(input_graph="hello")
 
     def test_n_bootstraps(self):
         for test in self.tests.keys():
             ldt = LatentDistributionTest(test, self.tests[test], n_bootstraps=123)
             ldt.fit(self.A1, self.A2)
             self.assertEqual(ldt.null_distribution_.shape[0], 123)
 
+    def test_passing_embeddings(self):
```

Comment:
wut happens if input_graph is False and a nx.graph gets passed?

Reply:
maybe add a test for this behavior (typeerror?)

Reply:
fur some reason thought that np.array(nx.Graph) breaks, but was surprised. agree that needs to be a test now.